indico/modules/events/registration/templates/emails/base_registration_details.html

```diff
@@ -43,7 +43,7 @@ <h2>{{ section.title }}</h2>
                 {{- friendly_data.choice }} ({{ friendly_data.arrival_date | format_date(locale='en_GB') }} - {{ friendly_data.departure_date | format_date(locale='en_GB') -}})
             {% endif %}
         {%- elif regdata.field_data.field.input_type == 'multi_choice' -%}
-            {{- friendly_data | join(', ') }}
+            {{- regdata.get_friendly_data(sort=False) | join(', ') }}
```

Comment:
```suggestion
            {{- regdata.get_friendly_data(sort=false) | join(', ') }}
```

in jinja the all-lowercase true/false/none is preferred